hudi-common/src/main/java/org/apache/hudi/common/model/FirstValueAvroPayload.java

```diff
@@ -27,6 +27,52 @@
 
 import java.util.Properties;
 
+/**
+ * Payload clazz that is used for Hudi Table.
+ *
+ * <p>Simplified FirstValueAvroPayload Logic:
+ * <pre>
+ *
+ *  Illustration with simple data.
+ *  the order field is 'ts', recordkey is 'id' and schema is :
+ *  {
+ *    [
+ *      {"name":"id","type":"string"},
+ *      {"name":"ts","type":"long"},
+ *      {"name":"name","type":"string"},
+ *      {"name":"price","type":"string"}
+ *    ]
+ *  }
+ *
+ *  case 1
+ *  Current data:
+ *      id      ts      name    price
+ *      1       1       name_1  price_1
+ *  Insert data:
+ *      id      ts      name    price
+ *      1       1       name_2    price_2
```

Comment:
`name_2    price_2` -> `name_2 price_2`

Reply:
done @danny0405  CI seem occur some oom error which is not related to the changed code